Sources/LiveViewNative/LiveViewNative.docc/Resources/01-01-03-config01.exs

```diff
@@ -0,0 +1,58 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+# General application configuration
+import Config
+
+# Configures the endpoint
+config :lvn_tutorial, LvnTutorialWeb.Endpoint,
+  url: [host: "localhost"],
+  render_errors: [
+    formats: [html: LvnTutorialWeb.ErrorHTML, json: LvnTutorialWeb.ErrorJSON],
+    layout: false
+  ],
+  pubsub_server: LvnTutorial.PubSub,
+  live_view: [signing_salt: "0p4irQ9b"]
+
+# Configure esbuild (the version is required)
+config :esbuild,
+  version: "0.17.11",
+  default: [
+    args:
+      ~w(js/app.js --bundle --target=es2017 --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
+    cd: Path.expand("../assets", __DIR__),
+    env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+  ]
+
+# Configure tailwind (the version is required)
+config :tailwind,
+  version: "3.2.7",
+  default: [
+    args: ~w(
+      --config=tailwind.config.js
+      --input=css/app.css
+      --output=../priv/static/assets/app.css
+    ),
+    cd: Path.expand("../assets", __DIR__)
+  ]
+
+# Configures Elixir's Logger
+config :logger, :console,
+  format: "$time $metadata[$level] $message\n",
+  metadata: [:request_id]
+
+# Use Jason for JSON parsing in Phoenix
+config :phoenix, :json_library, Jason
+
+config :live_view_native, :platforms, [LiveViewNativeSwiftUi.Platform]
+
+config :live_view_native, LiveViewNativeSwiftUi.Platform,
+  app_name: "LVNTutorial",
+  custom_modifiers: []
```

Comment:
Can you move these closer to the top of the file? The rendered tutorial doesn't do a great job indicating when there are changes below the fold, which can confuse people.